elements/lisk-tree/src/index.ts

```diff
@@ -14,3 +14,4 @@
 
 export { MerkleTree } from './merkle_tree/merkle_tree';
 export { verifyProof } from './merkle_tree/verify_proof';
+export { calculatePathNodes, calculateRootFromUpdateData } from './merkle_tree/calculate';
```

Comment:
`calculatePathNodes` does not need to be exported

Reply:
done